packages/manager/modules/dedicated-universe-components/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ovh-ux/ng-ovh-dedicated-universe-components",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud DedicatedUniverseComponents",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/dedicated-universe-components"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
```

Comment:
you are using rollup but the main entry of you package json is src/index.js

either not use rollup at all or make the main point the the dist/esm/index.js

Reply:
I see identical code in cloud-universe-components, targeted to ./src/index.js 
Should I change in both places?

Reply:
what do you mean ?
the main entry of cloud-universe-components is pointing to the dist : https://github.com/ovh/manager/blob/master/packages/manager/modules/cloud-universe-components/package.json#L23

Reply:
I changed it to ./dist/esm/index.js